pokete.py

```diff
@@ -1246,48 +905,49 @@ def game(_map):
                     save()
                     exiter()
             elif _ev.get() == "':'":
                 _ev.clear()
-                inp = text_input(movemap.code_label, movemap, ":",
-                                 movemap.width,
-                                 (movemap.width - 2) * movemap.height - 1)[1:]
-                movemap.code_label.outp(figure.map.pretty_name)
+                inp = text_input(mvp.movemap.code_label, mvp.movemap, ":",
+                                 mvp.movemap.width,
+                                 (mvp.movemap.width - 2) * mvp.movemap.height - 1)[1:]
+                mvp.movemap.code_label.outp(figure.map.pretty_name)
                 codes(inp)
                 _ev.clear()
         std_loop()
         _map.extra_actions()
+        Meadow.moving_grass(all_gras_objs)
         time.sleep(0.05)
-        for statement, x, y in zip([figure.x + 6 > movemap.x + movemap.width,
-                                    figure.x < movemap.x + 6,
-                                    figure.y + 6 > movemap.y + movemap.height,
-                                    figure.y < movemap.y + 6],
+        for statement, x, y in zip([figure.x + 6 > mvp.movemap.x + mvp.movemap.width,
+                                    figure.x < mvp.movemap.x + 6,
+                                    figure.y + 6 > mvp.movemap.y + mvp.movemap.height,
+                                    figure.y < mvp.movemap.y + 6],
                                    [1, -1, 0, 0], [0, 0, 1, -1]):
             if statement:
-                movemap.set(movemap.x + x, movemap.y + y)
+                mvp.movemap.set(mvp.movemap.x + x, mvp.movemap.y + y)
         # checking for resizing
         width, height = os.get_terminal_size()
-        if movemap.width != width or movemap.height != height - 1:
-            movemap.resize(height - 1, width, " ")
-        movemap.full_show()
+        if mvp.movemap.width != width or mvp.movemap.height != height - 1:
+            mvp.movemap.resize(height - 1, width, " ")
+        mvp.movemap.full_show()
 
 
 def intro():
     """Intro to Pokete"""
-    movemap.set(0, 0)
-    movemap.bmap = ob_maps["intromap"]
-    movemap.full_show()
+    mvp.movemap.set(0, 0)
+    mvp.movemap.bmap = obmp.ob_maps["intromap"]
+    mvp.movemap.full_show()
     while figure.name in ["DEFAULT", ""]:
-        figure.name = ask_text(movemap,
+        figure.name = ask_text(mvp.movemap,
                                "Welcome to Pokete!\nPlease choose your name!\n",
                                "Name:", "", "Name", 17)
-    movemap.name_label_rechar(figure.name)
-    movemap.text(4, 3, [" < Hello my child.",
+    mvp.movemap.name_label_rechar(figure.name)
+    mvp.movemap.text(4, 3, [" < Hello my child.",
                         " < You're now ten years old.",
                         " < And I think it's now time for you to travel the \
 world and be a Pokete-trainer.",
```

Comment:
Due to adding `mvp.` to line 950, the indentation in these lines changes, which makes them too long:

```suggestion
                            " < You're now ten years old.",
                            " < And I think it's now time for you to travel \
the world and be a Pokete-trainer.",
```

PS: Please also fix indentation of lines 954-968, as GitHub doesn't let me make a suggestion in these lines